app/graphql/types/subscriptions/object.rb

```diff
@@ -17,7 +17,8 @@ class Object < Types::BaseObject
       field :billing_time, Types::Subscriptions::BillingTimeEnum
       field :canceled_at, GraphQL::Types::ISO8601DateTime
       field :ending_at, GraphQL::Types::ISO8601DateTime
-      field :on_termination_credit_note, Types::Subscriptions::OnTerminationCreditNoteEnum, null: true
+      field :on_termination_credit_note, Types::Subscriptions::OnTerminationCreditNoteEnum
+      field :on_termination_invoice, Types::Subscriptions::OnTerminationInvoiceEnum, null: false
```

Comment:
I always find it confusing to know if it's nullable by default 😄

Reply:
That's exactly what I was thinking when writing this. This should be a Rubocop rule to be honest.